basics.py

```diff
@@ -0,0 +1,115 @@
+import pandas as pd 
+import matplotlib.pyplot as plt
+import numpy as np
+from pandas_datareader import data as web
+from scipy import stats
+import seaborn as sns
+from datetime import datetime
+
+pd.options.display.float_format = '{:.5f}'.format
+
+def get_data_close(symbol, start, end):
+    df = pd.DataFrame(web.DataReader(symbol, data_source='yahoo', start = start, end = end)['Adj Close'])
```

Comment:
Since we're using these in the [final library](https://github.com/stocksmith/stocksmith/) which is meant to deliver abstract functions to the user, it might make sense to let any constants be variables. 

For instance, the data that the client uses might have another term for `Adj Close`, which would break their application if they're using our packages.

It's fine for now, but we'll need to start thinking about abstraction early on.